content/about/introduction/introduction.html

```diff
@@ -14,166 +14,161 @@
+          APGのガイダンスの一部は規範的な仕様によってカバーされていません。
+          例えば、パターンには、ARIAによって指定された規範的な要件と、一般的なオペレーティングシステムおよびソフトウェアによって確立された規約を組み合わせたキーボードインタラクションモデルが含まれています。
+          <a class="specref" href="#managingfocus">フォーカスの管理</a>のセクションでは、ARIAには特定のタイプのウィジェット内で<kbd>Tab</kbd>キーがフォーカスを移動させないようにするという規範的な言語が含まれています。
+          しかし、ARIAは、これらのウィジェット内で特定のキーがどのようにフォーカスを移動させるべきかを指定していません。
+          そのため、APGタスクフォースは、広く利用されているキーボードの規約に依存して、ナビゲーションおよび操作のためのキーボードコマンドのリストを各パターンに提供します。
+          これらのリストは規範的ではありませんが、そのような規約の大部分は非常によく確立されているため、要件として扱うことが望ましいです。
+          その場合でない場合、ガイダンスは明確にオプショナルとしてマークされています。
+          注記として、ウェブウィジェットがユーザーのオペレーティングシステムの規約に動的に適応するように調整された場合、ユーザーにとってより良いサービスが提供される可能性があるキーボードコマンドがいくつかあります。
+          一部のAPGキーボードインタラクションモデルがmacOS固有のガイダンスを十分に提供していないというのは<a href="https://github.com/w3c/aria-practices/issues/1438">既知の問題</a>です。
+        </p>
       </section>
 
       <section id="caution-when-deviating">
-        <h2>Exercise Caution If Deviating from APG Patterns</h2>
+        <h2>APGパターンから逸脱する場合の注意</h2>
         <p>
-          While APG is not a standard, its patterns and practices do convey requirements from normative standards.
-          To help authors address a wide variety of UI design needs, the patterns and practices describe ranges of possibilities and options that satisfy those standards.
-          The examples each represent a specific way of implementing one or more patterns; they are intended to be illustrative, not prescriptive.
+          APGは標準ではありませんが、そのパターンと実践は規範的な標準からの要件を伝えます。
+          著者がさまざまなUIデザインのニーズに対処するのを助けるために、パターンと実践はそれらの標準を満たす可能性のある範囲とオプションを説明します。
+          各例は、1つ以上のパターンを実装する特定の方法を代表しており、規範的であることを意図していません。
         </p>
         <p>
-          It is important to understand, however, that the APG pattern library is not intended to serve as a catalog of all possible, valid, and potentially usable ways of building accessible user interfaces.
-          Consequently, some developers may find it necessary to go beyond  simply combining various APG patterns to create an experience that supports their UI requirements.
-          If building UI that deviates from APG patterns, ensure the design and behavior of the UI:
+          しかし、APGパターンライブラリは、アクセシブルなユーザーインターフェースを構築するためのすべての可能で、有効で、おそらく使用可能な方法のカタログとして機能することを意図しているわけではありません。
+          そのため、一部の開発者は、UI要件をサポートする体験を作成するために、単にさまざまなAPGパターンを組み合わせることを超えて進む必要があると感じるかもしれません。
+          APGパターンから逸脱するUIを構築する場合は、UIのデザインと動作が以下と一致していることを確認してください：
         </p>
         <ul>
-          <li>Are consistent with relevant guidance covered by <a href="../../practices/practices.html">APG practices</a>.</li>
-          <li>Comply with normative requirements of the <a class="specref" href="#abstract">ARIA specification</a>.</li>
-          <li>Follow or mimic keyboard conventions in similar APG patterns or native operating system widgets.</li>
+          <li><a href="../../practices/practices.html">APG実践</a>によってカバーされている関連するガイダンスと一致している。</li>
```

Comment:
一致というより一貫性がだいじなのかなと！

```suggestion
          <li><a href="../../practices/practices.html">APG実践</a>によってカバーされている関連するガイダンスと一貫性があること。</li>
```